src/nnfusion/core/kernels/cuda_gpu/cuda_langunit.hpp

```diff
@@ -13,15 +11,15 @@ namespace nnfusion
             LU_DECLARE(cuda);
             LU_DECLARE(cublas);
             LU_DECLARE(cudnn);
-            LU_DECLARE(super_scaler);
+            LU_DECLARE(superscaler);
             LU_DECLARE(cupti);
             LU_DECLARE(cuda_prof_api);
-            LU_DECLARE(cuda_fp16);
+						LU_DECLARE(cuda_fp16);
```

Comment:
u might need to run `make style` inside ur build dir to format the changes. otherwise code style pass check will fail